LocalCompare 'updates the reference image...' test fails

I see the following error when running the unit tests:

``` sh
1) LocalCompare afterScreenshot updates the reference image when changes are in tolerance:
     AssertionError: File should be modified: expected 1466169655000 to be above 1466169655000
```

Comparing the stat calls the files appear identical:

``` sh
first run - create reference file
statsFirst { dev: 16777223,
  mode: 33188,
  nlink: 1,
  uid: 501,
  gid: 20,
  rdev: 0,
  blksize: 4096,
  ino: 30557209,
  size: 293,
  blocks: 8,
  atime: Fri Jun 17 2016 14:20:55 GMT+0100 (BST),
  mtime: Fri Jun 17 2016 14:20:55 GMT+0100 (BST),
  ctime: Fri Jun 17 2016 14:20:55 GMT+0100 (BST),
  birthtime: Fri Jun 17 2016 14:20:55 GMT+0100 (BST) }
Image is within tolerance or the same. Updating base image
statsSecond { dev: 16777223,
  mode: 33188,
  nlink: 1,
  uid: 501,
  gid: 20,
  rdev: 0,
  blksize: 4096,
  ino: 30557209,
  size: 293,
  blocks: 8,
  atime: Fri Jun 17 2016 14:20:55 GMT+0100 (BST),
  mtime: Fri Jun 17 2016 14:20:55 GMT+0100 (BST),
  ctime: Fri Jun 17 2016 14:20:55 GMT+0100 (BST),
  birthtime: Fri Jun 17 2016 14:20:55 GMT+0100 (BST) }
```

--- a/test/unit/methods/SaveScreenshot.test.js
+++ b/test/unit/methods/SaveScreenshot.test.js
@@ -97,7 +97,7 @@ describe('SaveScreenshot', function () {
       assert.isAbove(statsFirst.mtime.getTime(), 0);
 
       // wait to get a different last modified time
-      await pause(100);
+      await pause(1000);
 
       // 2nd run --> update reference image
       const resultSecond = await this.saveScreenshot.afterScreenshot(context, base64Screenshot);
@@ -138,7 +138,7 @@ describe('SaveScreenshot', function () {
       assert.isAbove(statsFirst.mtime.getTime(), 0);
 
       // wait to get a different last modified time
-      await pause(100);
+      await pause(1000);
 
       // 2nd run --> update refernece with diff image
       const resultSecond = await this.saveScreenshot.afterScreenshot(context, base64ScreenshotNew);

